### PR TITLE
Code for a adding two matrix

### PR DESCRIPTION
**Pull request by shahabrar7746**
This code will be used to add two matrix in other words it can add two 2D arrays

### **Describe your change:**

- [*] Add an algorithm?
- [ ] Fix a bug or typo in an existing algorithm?
- [ ] Documentation change?

### **Checklist:**

- [ ] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [*] This pull request is all my own work -- I have not plagiarized.
- [*] I know that pull requests will not be merged if they fail the automated tests.
- [*] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [ ] All new Java files are placed inside an existing directory.
- [ ] All filenames are in all uppercase characters with no spaces or dashes.
- [ ] All functions and variable names follow Java naming conventions.
- [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
- [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: DataStructures/matrix_Add.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
+
```
Comment (anantcodes) [inline DataStructures/matrix_Add.java:1]: Please remove extra spaces.

**Review by anantcodes — COMMENTED** [review]
Add some test cases and remove extra spaces.

**Comment by shahabrar7746** [comment]
@anantcodes  hey glad you reviewed my PR could help me a little bit

**Comment by anantcodes** [comment]
Checkout other PRs that were successfully merged.

**Pull request closed without merging**